test/core/rules/docstring_test.py

```diff
@@ -0,0 +1,37 @@
+"""Test rules docstring"""
+import pytest
+
+from sqlfluff.core.plugin.host import get_plugin_manager
+
+KEYWORD_ANTI = "\n    | **Anti-pattern**"
+KEYWORD_BEST = "\n    | **Best practice**"
+KEYWORD_CODE_BLOCK = "\n    .. code-block:: sql\n"
```

Comment:
Do we need the white space here? Seems fragile.

```suggestion
KEYWORD_ANTI = "**Anti-pattern**"
KEYWORD_BEST = "**Best practice**"
KEYWORD_CODE_BLOCK = ".. code-block:: sql\n"
```

Reply:
@tunetheweb I add space to avoid the indentation issue for docstring, and I do not have strong opinion here

Reply:
I change my mind, I think white space have to add here, to ensure rule reference have same indent.
```diff
    | **Anti-pattern**
    | The • character represents a space.

    .. code-block:: sql

        SELECT
            a
        FROM foo••

    | **Best practice**
    | Remove trailing spaces.

-.. code-block:: sql
-
-    SELECT
-         a
-    FROM foo
+    .. code-block:: sql
+
+        SELECT
+            a
+        FROM foo
```
without white space, the red part would pass test but what we need is the green part

Reply:
OK. Was just concerned if we ever changed the indentation in future. But guess it's a good check until then. Resolving.